### PR TITLE
feat: Add logic to re caculate route for sending a transaction based on networks disabled by the user

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
Add logic to re calculate route for sending a transaction based on networks disabled by the user

### CODE REVIEW

File: services/wallet/api.go [hunk 1]
```diff
@@ -318,9 +318,9 @@ func (api *API) GetSuggestedFees(ctx context.Context, chainID uint64) (*Suggeste
 	return api.s.feesManager.suggestedFees(ctx, chainID)
 }
 
-func (api *API) GetSuggestedRoutes(ctx context.Context, account common.Address, amount float64, tokenSymbol string) (*SuggestedRoutes, error) {
+func (api *API) GetSuggestedRoutes(ctx context.Context, account common.Address, amount float64, tokenSymbol string, disabledNetworks []string) (*SuggestedRoutes, error) {
```
Comment (alaibe) [inline services/wallet/api.go]: could you maybe keep same naming + type as in other methods:
```disabledNetworks []string```
become:
```
disabledChainIDs []uint64
```

**Review by alaibe — APPROVED** [review]
Approving to unblock, but comment about disabledChainIDs is still valid

File: services/wallet/router.go [hunk 2]
```diff
@@ -71,6 +72,23 @@ func (r *Router) suggestedRoutes(ctx context.Context, account common.Address, am
 			continue
 		}
 
+		networkFound := false
+		for _, disabledNetwork := range disabledNetworks {
+			networkFound = false
+			uintVar, err := strconv.ParseUint(disabledNetwork, 10, 64)
```
Comment (alaibe) [inline services/wallet/router.go]: why not passing int directly?

**Comment by Khushboo-dev-cpp** [comment]
resolved the review comments @alaibe :)

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**